wikipedia-dark.user.css

```diff
@@ -1490,13 +1840,19 @@ regexp("https?:\/\/wikiless\.org\/.*$") {
   .mwe-popups .mwe-popups-container .mwe-popups-thumbnail {
     background-color: var(--white) !important;
   }
-  span[style="color:black"], span[style*="color: black"],
-  span[style*="color:black"], td[style*="color: black"],
-  td[style*="color: #2C2C2C"], .mwe-popups-container a {
+  span[style="color:black"],
+  span[style*="color: black"],
+  span[style*="color:black"],
```

Comment:
one of these can be removed, span[style*="color:black"] will match span[style*="color:black"] so keep the span[style*="color:black"]